TECHNICAL_CHARTER.md

```diff
@@ -10,13 +10,13 @@ This charter (the "Charter") sets forth the responsibilities and procedures for
 
     b. The scope of the Project includes collaborative development under the Project License (as defined herein) supporting the mission, including documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open source project.
 
-2. **Technical Steering Committee**
+2. **Community Governance Committee**
 
-    a. The Technical Steering Committee (the "TSC") will be responsible for all technical oversight of the open source Project.
+    a. The Community Governance Committee (the "CGC") will be responsible for all technical oversight of the open source Project.
 
-    b. The TSC voting members are initially the Project's Committers. At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project's code repository. The TSC may choose an alternative approach for determining the voting members of the TSC, and any such alternative approach will be documented in the CONTRIBUTING file. Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person.
+    b. The CGC voting members are initially the Project's Committers. At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project's code repository. The CGC may choose an alternative approach for determining the voting members of the CGC, and any such alternative approach will be documented in the CONTRIBUTING file. Any meetings of the Community Governance Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person.
```

Comment:
I did not think that CGC meetings were meant to be open to the public(although the previous text does state the same). Is that the intent?

Reply:
Yes, that is indeed the intent. As an openly developed, collaborative,  open source project, the meetings of the TSC/CGC are intended to be public.

This does not preclude _some_ meetings from being held with a limited audience, e.g., to handle discussions around Code of Conduct or security vulnerabilities. It merely requires that the regular, recurring meetings of the TSC/CGC be open to all.